packages/babel-plugin-transform-runtime/src/index.js

```diff
@@ -1,10 +1,31 @@
+import path from "path";
+import resolve from "resolve";
 import { declare } from "@babel/helper-plugin-utils";
 import { addDefault, isModule } from "@babel/helper-module-imports";
 import { types as t } from "@babel/core";
 
 import definitions from "./definitions";
 
-export default declare((api, options) => {
+function resolveAbsoluteRuntime(moduleName: string, dirname: string) {
+  try {
+    return path.dirname(
+      resolve.sync(`${moduleName}/package.json`, { basedir: dirname }),
```

Comment:
Why do we need to look for package.json?

Reply:
The runtime doesn't have a `main` or `index` because it's meant to be a collection of independent files. Since we want to resolve the directory location though, we do need a specific file to resolve, and `package.json` fits those requirements well.